feat(sig-services): Create data collection/permission doc (#118)
open-services-group/community: sig-services/service-requirements/data-collection-permission.md

---

> Title: Data Collection/Permission

Reviewer: ```suggestion
Title: Data Collection and Permissions
```

> Description: social login

Reviewer: ```suggestion
```

Persona: service consumer, service provider/owner
Phase: service-operational
---

> This requirement states that a service should have a well defined social login description. Social login means the kind of information one gets from the user when one login to the serivce.

Reviewer: https://www.techtarget.com/whatis/definition/social-login

> This requirement states that a service should have a well defined social login description. Social login is a single sign-on (SSO) technology that allows users to authenticate themselves on various applications and sites by connecting through social media site rather then typing a separate ID and password on each website.

Reviewer 2: ```suggestion
If service relies on social login for user identity it should describe why and what information it collects from third party identity providers. Social login is a single sign-on (SSO) technology that allows users to authenticate themselves on various applications and sites by connecting through social media site rather then typing a separate ID and password on each website.
```

Author: Thank you @tumido for the feedback. I have made the changes as suggested.

It should also clearly state the reason behind getting the data priviledges in order to impersonate on their behalf.

## Requirements

> - Service owner should be subscribed to the service.

Reviewer: This guideline should focus on what content goes in a data collection and permissions doc. So points about how can we be clear to the service consumer about their data that we'll collect. Are we collecting their github id and name? Their Github org perms? Their browser session info? This guideline should say that if we are doing that then we should mention it in a doc.

Author: Thanks for the suggestions @Shreyanand . I have tried to update the doc. Lmk your feedback. :)
